lang/cs/Org.Apache.REEF.Common/Org.Apache.REEF.Common.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Hi,
We are using the *.DotNet projects, so these changes need to go into Apache.REEF.Common.DotNet.csproj. Sorry for the confusion. I have a PR out that deletes the old projects but it hasn't gone through yet.

Reply:
Actually if you only added cs files to the project, then no changes should be needed since the DotNet.csproj should automatically pickup the new files.

Reply:
Double checked and everything seems to be in place in the Reef.DotNet.sln too 👍